HISTORY.md

```diff
@@ -7,7 +7,9 @@
 * Change BlockBasedTableOptions.cache_index_and_filter_blocks_with_high_priority's default value from false to true.
 
 ### Public API Change
-* Index, filter, and compression dictionary blocks are now handled similarly to data blocks with regards to the block cache: instead of storing objects in the cache, only the blocks themselves are cached. In addition, index, filter, and compression dictionary blocks (as well as filter partitions) no longer get evicted from the cache when a table is closed. Moreover, index blocks can now use the compressed block cache (if any), and cached index blocks can be shared among multiple table readers.
+* Filter and compression dictionary blocks are now handled similarly to data blocks with regards to the block cache: instead of storing objects in the cache, only the blocks themselves are cached. In addition, filter and compression dictionary blocks (as well as filter partitions) no longer get evicted from the cache when a table is closed.
+* Due to the above refactoring, block cache eviction statistics for filter and compression dictionary blocks are temporarily broken. We plan to reintroduce them in a later phase.
+* The semantics of the per-block-type block read counts in the performance context now match those of the generic block_read_count.
```

Comment:
This line is already in HISTORY.md file in branch 6.3.fb branch. Should we fix there too? Btw, line 14, 15 and 16 are also in HISTORY.md file there. Either this file is wrong or branch 6.3.fb is wrong.

Reply:
Hm, I don't see it on 6.3.fb: https://github.com/facebook/rocksdb/blob/6.3.fb/HISTORY.md